cdist/conf/type/__jail_freebsd10/gencode-remote

```diff
@@ -36,7 +36,7 @@ state="$(cat "$__object/parameter/state")"
 
 started="true"
 # If the user wants the jail gone, it implies it shouldn't be started.
-[ -f "$__object/parameter/stopped" -o "$state" = "absent" ] && started="false"
+([ -f "$__object/parameter/stopped" ] || [ "$state" = "absent" ]) && started="false"
```

Comment:
we can avoid the sub-shell by using command grouping `{ true; false; }`:

    { [ -f "$__object/parameter/stopped" ] || [ "$state" = "absent" ]; } && started="false"

 (there are other occurrences of this)